This test fail on some of our debian 13 test machine and golang 1.17 is used only
to build more recent golang versions
---
 src/syscall/exec_linux_test.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/syscall/exec_linux_test.go b/src/syscall/exec_linux_test.go
index d62b80b017..2b32ea7f7d 100644
--- a/src/syscall/exec_linux_test.go
+++ b/src/syscall/exec_linux_test.go
@@ -337,6 +337,8 @@ func TestUnshareMountNameSpaceHelper(*testing.T) {
 // Test for Issue 38471: unshare fails because systemd has forced / to be shared
 func TestUnshareMountNameSpace(t *testing.T) {
 	skipInContainer(t)
+	t.Skip("skipping: not supported in SlapOS")
+
 	// Make sure we are running as root so we have permissions to use unshare
 	// and create a network namespace.
 	if os.Getuid() != 0 {
-- 
2.52.0

